actions/src/project_actions.py

```diff
@@ -127,6 +152,7 @@ def project_update(
         email: str,
         description: str,
         is_enabled: str,
+        immutable: str,
```

Comment:
I thought Stackstorm passed through bool types rather that us having to do the comparison if we set the correct type in the action layer?

Reply:
I made it a drop down of true, false, unchanged along with the enabled property so we can leave them as they are rather than always updating them every time the action is run to update the other properties like email.